csrc/scheduler/vectorize_helper.cpp

```diff
@@ -997,22 +998,22 @@ int64_t getVectorizationFactor(
 
   const auto& resize_factors = resize_factors_entry.get();
 
-  int64_t max_vec_size = SchedulerRuntimeInfo::max_alignment_size_in_bit;
+  int64_t max_vect_factor = max_vectorization_size_in_bit;
```

Comment:
Why not just call `SchedulerRuntimeInfo::getMaxVectorizationSizeInBit`?

Reply:
Same to the previous question. Want to let each scheduler controls the max vectorization bits. We can change to directly use `SchedulerRuntimeInfo::getMaxVectorizationSizeInBit` after all the schedulers are checked.